icc-x-api/icc-patient-x-api.ts

```diff
@@ -499,11 +517,12 @@ export class IccPatientXApi extends iccPatientApi {
             AES.importKey("raw", utils.hex2ua(sfks.extractedKeys[0].replace(/-/g, "")))
           )
           .then((key: CryptoKey) =>
-            AES.encrypt(key, utils.utf82ua(JSON.stringify({ note: p.note }))).then(es => {
-              p.encryptedSelf = btoa(utils.ua2text(es))
-              delete p.note
-              return p
-            })
+            utils.crypt(
```

Comment:
@cebamps @aduchate This means the full patient is encrypted or am I missing something?

Reply:
@antoinepairet no: the `IccPatientXApi` takes in a new constructor parameter which is an array of key names whose values will be encrypted. It defaults to just `["notes"]` so that this code change preserves default behaviour.